envs/environment.yaml

```diff
@@ -89,3 +89,4 @@ dependencies:
   - git+https://github.com/davide-f/google-drive-downloader@master  # google drive with fix for virus scan
   - chaospy  # lastest version only available on pip
   - fake_useragent
+  - currencyconverter
```

Comment:
Have there been any particular reasons to use `pip` distribution? As discussed, `pip` can easily lead to the dependencies conflicts and must be avoided

Reply:
As a follow-up of our discussion, a requirement to have `conda` installation is by no means ultimative and can be also managed on the maintenance side if there is no other way around 🙂